apps/financial-aid/backend/src/app/modules/municipality/municipality.service.ts

```diff
@@ -154,6 +161,14 @@
     currentUser: Staff,
   ): Promise<MunicipalityModel[]> {
     try {
+      if (municipality.navPassword) {
+        municipality.navPassword = CryptoJS.AES.encrypt(
+          municipality.navPassword,
```

Comment:
Had a discussion with guðjón about this. It was decided to dismiss this warning because we have enough security by using a secret key and a random seed(initial vector) to encrypt.